### DISCUSSION TITLE
PLEASE Stop the Pop-Up on Query Execution Error!

### DISCUSSION THREAD

**Discussion by robarwebservices** [dbeaver/dbeaver#13401] [Q&A]
When a query returns an execution error, DBeaver throws a pop-up (in addition to displaying in the "output" tab).

This creates an extra unnecessary step of closing the dialog. It's a pain, and it creates unnecessary friction when trying to get work done. 

I don't see an option to change it. Please give us this option to disable the execution error pop up!

![image](https://user-images.githubusercontent.com/334314/121204876-97380500-c84d-11eb-8656-887446489141.png)

**Comment by uslss** [comment]
please, uncheck Show errors in dialog option in Preferences
![image](https://user-images.githubusercontent.com/31996417/121303780-26075900-c904-11eb-8ab4-cfdd2f6e1ae4.png)

**Comment by robarwebservices** [comment]
It IS unchecked by default. This isn't the right setting. I still get the pop-up on query execution error. I'm on the latest version for windows.

**Comment by uslss** [comment]
is it still actual?

**Comment by robarwebservices** [comment]
Yes!

**Comment by uslss** [comment]
still cannot reproduce it. which dbeaver version do you use?
![image](https://user-images.githubusercontent.com/31996417/124094486-7f752a80-da61-11eb-9aff-d67d4cf8dd90.png)

**Comment by c-o-m-m-a-n-d-e-r-d-a-t-a** [comment]
I currently have the opposite problem - ie I can't get it to turn on. I want to see the errors, and they used to pop up, but now they don't. I have tried the tick box mentioned above, but changing it has no effect.

**Reply by wheinen** [reply]
Were you ever able to fix this issue? I have the same problem, and made a new discussion for this: https://github.com/orgs/dbeaver/discussions/35460
